### PR TITLE
Enforce strict format for ID & name

### PR DESCRIPTION
**Pull request by smnandre**
(i'd put that on an Icon class, but this is another PR :angel:)

I really would like to
1. normalize things as early as possible
2. don't have multiple place where those checks are done, the main risk beeing they are not done..

Open to debate :)

### CODE REVIEW

**Comment by smnandre** [comment]
@kbond should i redo it for the draft-2 branch, or can we already discuss it ? As you want

File: src/Icons/src/Svg/SvgUtils.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+final class SvgUtils
+{
+    /**
+     * Transforms a valid icon ID into an icon name.
+     *
+     * @throws \InvalidArgumentException if the ID is not valid
+     * @see isValidId()
+     */
+    public static function idToName(string $id): string
```
Comment (kbond) [inline src/Icons/src/Svg/SvgUtils.php:13]: Could these be added to the Icon object in https://github.com/kbond/ux/pull/7?

Comment (kbond) [reply src/Icons/src/Svg/SvgUtils.php:13]: If so, then combine this PR with an updated #7.

Comment (smnandre) [reply src/Icons/src/Svg/SvgUtils.php:13]: Will do in a moment !

Comment (smnandre) [reply src/Icons/src/Svg/SvgUtils.php:13]: https://github.com/kbond/ux/pull/9

**Pull request closed without merging**